lldb/source/Plugins/Process/Linux/NativeRegisterContextLinuxArm64Shared.cpp

```diff
@@ -0,0 +1,63 @@
+#include "NativeRegisterContextLinuxArm64Shared.h"
+
+using namespace lldb_private::process_linux::arm64;
+
+namespace lldb_private {
+namespace process_linux {
+namespace arm64 {
```

Comment:
You don't need `using namespace` if you're then going to open the same namespaces.

Here I would just write out the fully qualified names and have `using namespace` if you still need it after doing that. `lldb_private::proccess_linux::arm64::ReadHardwareDebugInfo` for example.